agents-for-bedrock/use-case-examples/hr-assistant/agent/knowledge-base-assets/ParentalLeave.txt

```diff
@@ -0,0 +1,117 @@
+Policy: Paid Parental Leave - US
```

Comment:
how was this data generated?

Reply:
Data is getting changed to support public information from Amazon.com as approved by legal